jsign-crypto/src/test/resources/services/venafi-sign.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
The json files should contains the API responses, if I'm not mistaken this looks like a request.